db/seed.py

```diff
@@ -4,21 +4,41 @@
 import os
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from db.models import Base, Movie, Genre, Character, Line, Conversation
+from db.models import (
+    Character,
+    Conversation,
+    Genre,
+    Line,
+    Movie,
+)
 
-directory = os.path.abspath(os.getcwd())
-database_uri = 'sqlite:///' + os.path.join(directory, 'movie_dialogs.sqlite')
-# database_uri = 'postgresql://user:password@localhost:5432/database'
+DIRECTORY = os.path.abspath(os.getcwd())
+DATABASE_URI = 'sqlite:///' + os.path.join(DIRECTORY, 'movie_dialogs.sqlite')
+# DATABASE_URI = 'postgresql://user:password@localhost:5432/database'
 
+CORPUS_PATH = os.path.join(DIRECTORY, 'corpus')
+
+CHARACTERS_DATA = 'movie_characters_metadata.txt'
+CHARACTERS_MAPPING = ['id', 'name', 'movie_id', 'movie_title', 'gender', 'credit_pos']
+
+CONVERSATION_DATA = 'movie_conversations.txt'
+CONVERSATION_MAPPING = ['first_char_id', 'second_char_id', 'movie_id']
+
+LINE_DATA = 'movie_lines.txt'
+LINE_MAPPING = ['id', 'character_id', 'movie_id', 'character_name', 'text']
+
+MOVIE_DATA = 'movie_titles_metadata.txt'
+MOVIE_MAPPING = ['id', 'title', 'year', 'imdb_rating', 'num_imdb_votes']
 
 def get_data(path):
-    with open(path, encoding='latin-1') as f:
+    print(os.path.join(CORPUS_PATH, path))
+    with open(os.path.join(CORPUS_PATH, path), encoding='latin-1') as f:
         for line in f:
             yield line
 
-
```

Comment:
two blank lines before top-level functions and classes is style convention in python. i am not really familiar with pylint yet (my default linter is flake8), but i'm a bit surprised that it doesn't pick this up.